src/routers/resource.js

```diff
@@ -0,0 +1,22 @@
+const {Router} = require('express');
+const router = Router();
+const checkAuth = require('../middleware/auth');
+const checkRoleAuth = require('../middleware/role-auth');
+const {postImageAccount, postImageLesson, deleteResource, patchResource, postAudio} = require('../controllers/resource-controller');
+const {validationURL} = require('../validators/account-validator');
+const {validateExistsAccount} = require('../controllers/account-controller');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8292860)
```suggestion
const {validateExistsAccount} = require("../controllers/account-controller");
```